lib/upipe-modules/upipe_rtcp_fb_receiver.c

```diff
@@ -0,0 +1,668 @@
+/*
+ * Copyright (C) 2016-2017 Open Broadcast Systems Ltd
+ *
+ * Authors: Rafaël Carré
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
```

Comment:
If you want LGPL you cannot put it in upipe-modules

Reply:
@kierank